app/controllers/clearance_batches_controller.rb

```diff
@@ -7,17 +7,16 @@ def index
    end
 
   def create
-    clearance_batch = ClearanceBatch.create
     potential_item_ids = process_file(params[:csv_batch_file].tempfile)
-    clearance_batch.clearance_items!(potential_item_ids)
+    service = ClearancingService.new
+    service.process_items(potential_item_ids)
```

Comment:
Since our processing method returns the status, we could condense these lines to:
```
clearancing_status = ClearancingService.new.process_items(potential_item_ids)
```
so that we wouldn't have to continue referencing the service object.